model/src/lib.rs

```diff
@@ -86,9 +89,9 @@ impl OrderCreation {
 
 // Intended to be used by tests that need signed orders.
 impl OrderCreation {
-    pub const TEST_DOMAIN_SEPARATOR: [u8; 32] = [0u8; 32];
+    pub const TEST_DOMAIN_SEPARATOR: DomainSeparator = DomainSeparator([0u8; 32]);
```

Comment:
nit: isn't this just DomainSeparator::Default?

Reply:
The compiler yells at me::
```
calls in constants are limited to constant functions, tuple structs and tuple variants
```
So I have to use lazy_static, and I guess the current solution is shorter and easier to read.